code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -552,13 +552,22 @@
 	return H
 
 /mob/living/silicon/pai/MouseDrop(atom/over_object)
-	var/mob/living/carbon/H = over_object
-	if(!istype(H) || !Adjacent(H)) return ..()
-	if(H.a_intent == I_HELP)
-		get_scooped(H)
-		//return
-	else
-		return ..()
+  var/mob/living/carbon/H = over_object
+  if(!istype(H) || !Adjacent(H))  return ..()
+  if(usr == src)

```

Comment:
MouseDrop doesn't have a user var. I'm not sure there's anything else I could use.